wiki/Interface/zh.md

```diff
@@ -1,47 +1,49 @@
----
-outdated: true
----
+# 界面
 
-# Interface (中文)
-
-![](img/Intro_static.jpg "Introduction screen")
+![](img/Intro_static.jpg "初始屏幕")
 
 ## 主菜单
 
-![](img/Introscreen.jpg "Main menu")
+![](img/Introscreen.jpg "主菜单")
 
-［1］osu！的标志会随着音乐的每分钟击打次数（BPM）跳动，如果没有在播放歌曲，osu！标志就会跳动得慢一点。下列在括弧里的是快捷键。
+[osu!粉饼](/wiki/Glossary#cookie) \[1\]随着 [BPM](/wiki/Beatmapping/Beats_per_minute) 跳动，此外，从osu!粉饼延伸出来的条的长度与歌曲的音量一致。如果没有在播放歌曲或者暂停播放，osu!粉饼就会以60BPM的速度跳动。主菜单的元素如下：
 
-- ［2］点击 Play（P）或者 osu! 标志进入次级选择界面。
-- ［3］点击 Edit（E）进入地图编辑器的歌曲选择界面。
-- ［4］点击 Option（O）进入设置界面。
-- ［5］点击 Exit（esc）退出 osu!
-- ［6］在主菜单下方会有随机的游戏提示。
-- ［7］左下角是 osu! 官网的链接。
-- ［8］与 Bancho 的链接状态。
-- ［9］右下方是聊天面板（F8&F9）。
-- ［10］右上角是随机播放的播放器。
-- ［11］用户信息旁边的是地图总数，运行时间，以及电脑时显示间。
-- ［12］用户面板，点击显示用户选项。
+- \[2\] 点击 Play （`P`） 或者 osu! 标志进入次级选择界面。
+- \[3\] 点击 Edit （`E`） 进入地图编辑器的歌曲选择界面。
+- \[4\] 点击 Option （`O`） 进入设置界面。
+- \[5\] 点击 Exit （`Esc`） 退出 osu!
+- \[6\] 在主菜单下方会有随机的游戏提示。
+- \[7\] 左下角是 osu! 官网的链接，以及版权信息。
+- \[8\] 与[Bancho](/wiki/Glossary#bancho)的连接状态，在这张图总没有显示，但是连接的结果应该像一个链条。
+- \[9\] 右下角的是扩展[聊天窗口](/wiki/Chat_Console)的控制开关 (叫做"玩家列表") 和常规聊天窗口 （分别对应 `F9` 和 `F8`）。
+- \[10\] 在右上角是osu!随机播放歌曲的控制器。顶部显示了当前正在播放的歌曲。从左到右，这些按钮的作用如下:
+  - 上一首
+  - 播放
+  - 暂停
+  - 停止 （与暂停不同的是，点击播放和停止会使歌曲从头播放，而暂停只是单纯的暂停）
+  - 下一首
+  - 浏览歌曲信息。 这个按钮会切换顶部歌曲信息是永久显示还是临时显示。当设置为永久显示时，歌曲信息将永久显示直到他与其他UI一起淡出。 当设置为临时显示时，他将会在歌曲被选择后一段时间消失直到被状态切换或者播放另一首歌时被唤出。
+- \[11\] 你拥有的谱面数量，osu!的运行时间，和你的系统时间。
+- \[12\] 用户面板，点击显示用户选项。
 
 ### 用户选项
 
-![](img/Useroptions1.png "User Options")
+![](img/Useroptions1-ZH.png "用户选项")
 
 点击屏幕左上角的用户面板以打开用户选项。此时将无法使用聊天界面。点击相应的选项以完成操作。
 
-1. **View Profile**：在浏览器中打开玩家主页
-2. **Sign Out**：退出登录
-3. **Change Avatar**：在浏览器中打开更改头像页面
-4. **Close** ：关闭此窗口
+1. `查看个人资料`：在浏览器中打开玩家主页。
+2. `注销`：退出登录 （退出登录后，[设置](/wiki/Options) 侧边栏会提醒你登录）。
+3. `修改头像`：在浏览器中打开更改头像页面。
+4. `关闭` ：关闭此窗口。
 
 ## 游戏界面
 
-![](img/Introscreen2.jpg "Play Menu")
+![](img/Introscreen2.jpg "游戏界面")
 
-- 点击 Solo（P）以进行单机游戏
-- 点击 Multi（M）以进行多人联机
-- 点击 Back 以返回主菜单
+- 点击 `Solo`（`P`）以进行单机游戏
+- 点击 `Multi`（`M`）以进行多人联机
```

Comment:
missing one sentence, original line:
```md
- Click `Multi` (`M`) to play with other people. You will be directed to the [Multi](/wiki/Multi) Lobby (see below).
```

Reply:
Sorry for forget that,
This sentence use the outdated text directly and forget to check.